src/akushon/action/node/action_manager.cpp

```diff
@@ -109,6 +109,10 @@ Action ActionManager::load_action(
           std::vector<Joint> joints;
 
           for (const auto & [joint_key, joint_val] : raw_pose["joints"].items()) {
+            if (strncmp(joint_key.c_str(), "neck", 4) == 0) {
```

Comment:
How about using [`std::string::compare`](https://en.cppreference.com/w/cpp/string/basic_string/compare) instead?

e.g.:
```suggestion
            if (joint_key.compare(0, 4, "neck") == 0) {
```